src/engine/blkstore/blkstore.hpp

```diff
@@ -325,6 +325,9 @@ class BlkStore {
     BlkAllocStatus alloc_contiguous_blk(const uint32_t size, blk_alloc_hints& hints, BlkId* const out_blkid) {
         // Allocate a block from the device manager
         assert(size % m_pagesz == 0);
+        static uint32_t max_suppoted_size = m_pagesz * std::numeric_limits< blk_count_t >::max();
```

Comment:
nit:  ideally this `max_supported_size` should come from 1Chunk::BlkAlloc::max_supported_size()`. We can do it from 4.x